include/opendht/def.h

```diff
@@ -41,4 +41,4 @@
 #endif // opendht_EXPORTS
 
 // bytes
-#define HASH_LEN 20u
+#define HASH_LEN 20u
```

Comment:
keep the trailing new line

Reply:
Done https://github.com/savoirfairelinux/opendht/pull/750/commits/1eba919dbd06b351d8a02aee63a6bcbb09af1210